### PR TITLE
[fix](iceberg) fix iceberg catalog rest access

### PR DESCRIPTION
**Pull request by wsjz**
# Proposed changes

Fix iceberg catalog access s3 
Fix iceberg catalog partition table query
Fix persistence

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.08 seconds
 load time: 498 seconds
 storage size: 17123367964 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230202135323_clickbench_pr_89569.html

File: docs/zh-CN/docs/lakehouse/multi-catalog/iceberg.md [hunk 1]
```diff
@@ -50,6 +52,47 @@ CREATE CATALOG iceberg PROPERTIES (
...
+);
+```
+
+若数据存放在S3上，properties中可以使用以下参数
+
+```sql
+"AWS_ACCESS_KEY" = "username"
+"AWS_SECRET_KEY" = "password"
+"AWS_REGION" = "region-name"
+"AWS_ENDPOINT" = "http://endpoint-uri"
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/iceberg.md:93]: Missing `AWS_CREDENTIALS_PROVIDER` config?
And missing English version

**Review by morningman — APPROVED** [review]
LGTM

File: docs/zh-CN/docs/lakehouse/multi-catalog/iceberg.md (hunk 1 above)
Comment (wsjz) [reply docs/zh-CN/docs/lakehouse/multi-catalog/iceberg.md:93]: use DataLakeAWSCredentialsProvider

**Pull request merged**